shopgate/pwa
# Updated Conductor in favor of a bug fix with resetting the router stack.

## Type of change

Please add an "x" into the option that is relevant:

- [ x ] Bug Fix :bug: (non-breaking change which fixes an issue)
- [  ] New Feature :rocket: (non-breaking change which adds functionality)
- [  ] Breaking Change :boom: (fix or feature that would cause existing functionality to not work as expected)
- [  ] Polish :nail_care: (Just some cleanups)
- [  ] Docs :memo: (Changes in the documentations)
- [  ] Internal :house: Only relates to internal processes.

## How to test it

Log in from homepage. Log out again. You should stay on the homepage and be able to log in again.
